app/src/main/java/de/westnordost/streetcomplete/data/QuestController.java

```diff
@@ -257,6 +267,53 @@ else if(quest.getStatus() == QuestStatus.CLOSED)
 		});
 	}
 
+	private boolean solveCompleteQuest(long questId, Bundle bundle)
+	{
+		CompleteQuest q = completeQuestDB.get(questId);
+
+		String answer;
+
+		switch (q.getComplete().completeType)
+		{
+			case CompleteTypes.CHART:
+				Boolean value = bundle.getBoolean(CompleteQuestType.ANSWER);
+				answer = value ? "Yes" : "No";
+				break;
+
+			case CompleteTypes.TRANSLATION:
+				answer = bundle.getString(CompleteQuestType.ANSWER);
+				break;
+
+			default:
```

Comment:
Also, in the corresponding issue another type was proposed: Answer with a picture. (You should be able to use the stuff from the notes there)

Reply:
Yes, that's true! Is it OK if the pictures would be uploaded to lut.im and then all shown in the web interface?

Reply:
Ask @ldidry (admin of lut.im), but based on the fact that he writes on the page you should not use it for Kodi playlists with hotlinking – and he does not complain about hotlinking there – I think hotlinking is ok. (Also, I'd say your evaluation website will not get so much viststs, and you do not have to load them instantly, but e.g. allow the user to click on a list or cycle through them, whatever… So you can keep the load low.)

Reply:
The problem with kodi's playlists is that thousands of people download 20, 30 or more pictures almost at the same time: after work, you go home and watch TV through Kodi (it's a guess, but that the only explanation I found to understand why I had so much requests from Kodi user-agent). The problem is not even the decryption of the images since lut.im has a Varnish cache, but the number of concurrent requests.

I remember being in touch with someone of this project, didn't I? That person asked if it was OK to use lut.im for image uploading. I said yes, since it would not be such a huge load: visitors will be far less than people watching kodi at the same time.

So OK for me.

If it was not someone from this project, it was OSM-related.

Please note that unlimited time storage has been deactivated a few months ago: the maximum time an image will be stored is one year.

Reply:
> I remember being in touch with someone of this project, didn't I?

Yes, that's right. We already have a feature implemented which uses your service and it works awesome! Thank you for providing it!

> Please note that unlimited time storage has been deactivated a few months ago: the maximum time an image will be stored is one year.

That's totally OK. I think one year is enough time to review some photos...

> So OK for me.

Wow cool! So I can work on implementing this!

Reply:
Great, thansk for your answer. Yes, you was likely in contact with the project maintainer @westnordost. But hey, asking on GitHub was so easy,

Reply:
This feature is now implemented on the client side but I have to do some work to display the data properly on the website... 🎉